app/javascript/components/froggo-team-metrics.vue

```diff
@@ -32,16 +44,40 @@ export default {
   beforeMount() {
     this.$store.dispatch(COMPUTE_TEAM_PR_INFORMATION, {
       githubUsers: this.froggoTeamMembers,
+      monthLimit: 3,
     });
   },
   components: {
     MetricsHandler,
+    ClickableDropdown,
   },
   props: {
     froggoTeamMembers: {
       type: Array,
       required: true,
     },
+    monthsOptions: {
+      type: Array,
+      default() {
+        return [];
+      },
+    },
+  },
+  data() {
+    return {
+      dropdownTitle: this.$t('message.profile.timespanDropdownTitle'),
```

Comment:
si se ocupa `dropdownTitle` solo para pasarselo a `clickable-dropdown` entonces ocuparia directamente `this.$t('message.profile.timespanDropdownTitle')`, o sea:

```
<clickable-dropdown
  ...otros props
  :body-title="this.$t('message.profile.timespanDropdownTitle')"
/>
```

Creo que eso debiese funcionar